src/redux/Admin/AddUser.actions.js

```diff
@@ -0,0 +1,37 @@
+export const ADD_USER_REQUEST = "ADD_USER_REQUEST"
+export const ADD_USER_SUCCESS = "ADD_USER_SUCCESS"
+export const ADD_USER_FAILURE = "ADD_USER_FAILURE"
+
+export const AddUserRequest = () => {
+  return {
+    type: ADD_USER_REQUEST
+  }
+}
+
+export const AddUserSuccess = (NewUser) => {
+  return {
+    type: ADD_USER_SUCCESS,
+    payload: NewUser
+   
+  }
+}
+
+export const AddUserFailure = (error) => {
+  return {
+    type: ADD_USER_FAILURE,
+    payload: error
+  }
+}
+
+export const AddUser = (id, NewUser) =>{
```

Comment:
There's an `isError` portion of the store [here](https://github.com/Lambda-School-Labs/frontend-vbb-portal/blob/main/src/redux/IsError.redux/IsError.reducer.js#L8) that you'll want to use in this flow. It helps us keep all the error management on the front end in one place. You can just dispatch the action from here rather than nest the error message inside of an object. It just helps with separation of concerns. 

You can see a pattern of that [here](https://github.com/Lambda-School-Labs/frontend-vbb-portal/blob/main/src/redux/Login.redux/Login.actions.js#L109-L142)

Reply:
OK--that is very helpful--thank you for that comment.